frontend/src/app/containers/HomePage.js

```diff
@@ -1,13 +1,9 @@
 import React from 'react';
 
 import HomePageNoAddon from './HomePageNoAddon';
-import HomePageWithAddon from './HomePageWithAddon';
 
 export default class HomePage extends React.Component {
   render() {
-    if (this.props.hasAddon) {
```

Comment:
With this logic gone, is there a difference between HomePage and HomePageNoAddon anymore? Could just delete one or the other.